src/Parser/Paladin/Retribution/Modules/Features/AlwaysBeCasting.js

```diff
@@ -47,17 +48,32 @@ class AlwaysBeCasting extends CoreAlwaysBeCasting {
     SPELLS.HAND_OF_RECKONING.id,
   ];
 
+  get suggestionThresholds() {
+    return {
+      actual: this.downtimePercentage,
+      isGreaterThan: {
+        minor: 0.1,
+        average: 0.2,
+        major: 0.3,
+      },
+      style: 'percentage',
+    };
+  }
+
   suggestions(when) {
     const deadTimePercentage = this.totalTimeWasted / this.owner.fightDuration;
+    const boss = this.owner.boss;
 
-    when(deadTimePercentage).isGreaterThan(0.1)
-      .addSuggestion((suggest, actual, recommended) => {
-        return suggest(<span>Your downtime can be improved. Try to Always Be Casting (ABC), try to reduce the delay between casting spells. Even if you have to move, try casting something instant with range like <SpellLink id={SPELLS.JUDGMENT_CAST.id} /> or <SpellLink id={SPELLS.DIVINE_STORM.id} /></span>)
-          .icon('spell_mage_altertime')
-          .actual(`${formatPercentage(actual)}% downtime`)
-          .recommended(`<${formatPercentage(recommended)}% is recommended`)
-          .regular(recommended + 0.1).major(recommended + 0.2);
-      });
+    if (!boss || !boss.fight.disableDowntimeSuggestion) {
+      when(deadTimePercentage).isGreaterThan(this.suggestionThresholds.isGreaterThan.minor)
+        .addSuggestion((suggest, actual, recommended) => {
```

Comment:
This can be updated to follow the style of the rest with `` when(deadTimePercentage).isGreaterThan(this.suggestionThresholds)``